### ISSUE TITLE
Underline as GIF shows

### ISSUE THREAD

**Issue by natematykiewicz** [w0rp/ale#1808]
You include this GIF in the README
![preview](https://raw.githubusercontent.com/w0rp/ale/master/img/example.gif)

How do you configure VIM to do that curly underline and also not break syntax highlighting? Notice that `let` stays blue, and doesn't turn orange like the curly underline is.

I've tried setting `ALEError` and `ALEWarning`, but can't seem to come close to what you have.

I tried running `:help ale-highlights` to see if it explains it, but I just get the following error: `E149: Sorry, no help for ale-highlights`.

**Comment by w0rp** [comment]
Use `silent! helptags ALL` to generate help tags, then you'll see the documentation. I recommend putting that in vimrc after loading the plugin in the README.

I've been using the [darkspectrum](https://github.com/vim-scripts/darkspectrum) color scheme for GVim for years now. You could try using that theme, or just some other theme.

**Comment by w0rp** [comment]
Yeah, I use GVim because the highlighting looks better than in terminals, among a few other things. I'll try and add a tooltip to the image explaining which theme is used.

**Comment by w0rp** [comment]
Okay, that should be there now.
